docs/specification/attribute-templates.md

```diff
@@ -177,7 +177,7 @@ Field name | Value type | Description
 -----------|------------|------------
 `type`     | String     | always `molecular_structure_bounds`
 
-##### Example
+## Example
```

Comment:
```suggestion
##### Example
```
It should maybe be `###`, but it should definitely be nested under `Molecular Structure Bounds`.  If this header is changed from `#####`, similar changes should be made to all the examples in each Bounds description.

The `##` header should be applied to line 189 (#### Example Attribute Templates) since that is properly nested under Attribute Templates and Bounds.